packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import MenuList from '@material-ui/core/MenuList';
+import Button from '@material-ui/core/Button';
+import MenuItem from '@material-ui/core/MenuItem';
+import { ApiContext } from '../api-context';
+import { useGridSelector } from '../../hooks/features/core/useGridSelector';
+import { optionsSelector } from '../../hooks/utils/optionsSelector';
+import { GridMenu } from '../menu/GridMenu';
+import { ExportOption } from '../../models';
+
+export function GridToolbarExport() {
+  const apiRef = React.useContext(ApiContext);
+  const options = useGridSelector(apiRef, optionsSelector);
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const ExportIcon = apiRef!.current.components!.ExportIcon!;
+
+  const ExportOptions: Array<ExportOption> = [
```

Comment:
I don't think we need that. And it is redefined at every render

Reply:
I think this is related to the discussion above it so I'll resolve this one to have it being discussed in a single place.